components/litetouch.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "LiteTouch",
+    "owner": ["patmann03"],
+    "manifest": "https://github.com/patmann03/custom_components/blob/master/custom_components/litetouch/manifest.json",
```

You need to point this to the raw url for the manifest.

```suggestion
    "manifest": "https://raw.githubusercontent.com/patmann03/custom_components/master/custom_components/litetouch/manifest.json",
```